### PR TITLE
recipe - search available POI lists by name

### PR DESCRIPTION
**Pull request by woozyking**
a recipe to simply search for available POI lists using a name string

```python
>>> search_avail_poi_lists('pizza')
                name  poi_list_id whitelabelid customerid
67       Bostonpizza           71         None       None
113       Pizzapizza          118         None       None
175  Neworleanspizza          180         None       None
422       Royalpizza          427         None       None
450       Ginospizza          455         None       None
786      Doublepizza          791         None       None
```

### CODE REVIEW

**Review by steven-mindswire — COMMENTED** [review]
maybe add case sensitive handler

File: recipes/search_avail_poi_lists.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+from locussdk import get_avail_poi_lists
+
+
+def search_avail_poi_lists(name):
```
Comment (maluhoss) [inline recipes/search_avail_poi_lists.py]: would recommend adding typehinting for parameters + a doc string so other users will know what this function + arguments mean like:

```python
from pandas import DataFrame

def search_avail_poi_lists(name: str) -> DataFrame:
"""
Retrieve available POI (Point of interest) lists based on the string provided
"""
```
Also recommend adding a default value for param so the last line does not fail if `name` is `False`

File: recipes/search_avail_poi_lists.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+    global_lists = get_avail_poi_lists(list_type='global')
+
+    return global_lists[global_lists['name'].str.contains(name)]
```
Comment (divyselva) [inline recipes/search_avail_poi_lists.py]: Including `case` parameter for `str.contains()` to add control for case sensitivity and broadening search scope 
i.e. `pizza` vs `Pizza`

File: recipes/search_avail_poi_lists.py [hunk 3]
```diff
@@ -0,0 +1,7 @@
+from locussdk import get_avail_poi_lists
+
+
+def search_avail_poi_lists(name):
+    global_lists = get_avail_poi_lists(list_type='global')
```
Comment (divyselva) [inline recipes/search_avail_poi_lists.py]: `list_type` can also be made possible for `custom`

**Pull request merged**